### PR TITLE
Update calls to simple_post to expect a requests.response

### PR DESCRIPTION
**Pull request by TylerSheffels**
This simplifies the request process here, and should resolve a very transient error: http://manage.dimagi.com/default.asp?189946#1065143

### CODE REVIEW

**Comment by TylerSheffels** [comment]
@benrudolph

File: dimagi/utils/post.py [hunk 1]
```diff
@@ -55,16 +56,9 @@ def simple_post(data, url, content_type="text/xml", timeout=60, headers=None):
...
-    up = urlparse(url)
-    if url.startswith("https"):
-        Connection = httplib.HTTPSConnection
-    else:
-        Connection = httplib.HTTPConnection
-    conn = Connection(up.netloc, timeout=timeout)
-    conn.request('POST', url, data, default_headers)
-    return conn.getresponse()
 
+    resp = requests.post(url, data, headers=default_headers)
```
Comment (benrudolph) [inline dimagi/utils/post.py]: nit: `return requests.post(url, data, headers=default_headers)`

**Comment by benrudolph** [comment]
yea :baby: much cleaner

**Comment by czue** [comment]
thank you. this is so much better. any QA concerns?

**Comment by TylerSheffels** [comment]
@czue I don't think there are tests covering this, I tried posting to a postbin, as well as to the endpoint mentioned in the ticket and both those were successful. I also examined the TCP packets, and this should only add more accurate headers to the request.  If there is a QA plan for this I would certainly prefer running it, I'll check with the QA team

**Comment by czue** [comment]
I doubt there is one. if you feel like you've tested sufficient cases i'm ok with merging but will let you make the call

**Comment by TylerSheffels** [comment]
Is there an easy way to see if the number of repeater failures goes up after the merge? or some way to monitor/soft assert this behaviour?

**Comment by benrudolph** [comment]
not that i know of but you could add datadog to the exception handler (where failure reason is set) by using this function: https://github.com/dimagi/commcare-hq/blob/master/corehq/util/datadog/utils.py#L54. datadog will automatically record any log item when using that

**Comment by TylerSheffels** [comment]
@benrudolph @czue I added some datadog logging that should note when the repeater record fails, and then I think this should be good to go.

**Comment by czue** [comment]
:+1:

**Pull request merged**